modules/@apostrophecms/translation/index.js

```diff
@@ -0,0 +1,220 @@
+module.exports = {
+  extend: '@apostrophecms/module',
+  options: {
+    enabled: true,
+    alias: 'translation'
+  },
+  init(self) {
+    self.providers = [];
+
+    self.enableBrowserData();
+  },
+  handlers(self) {
+    return {
+      '@apostrophecms/doc-type:beforeLocalize': {
+        // Translate the document using the first available provider.
+        async translate(req, doc, {
+          source, target, docType, existing
+        }) {
+          const targets = req.query.aposTranslateTargets || [];
```

Comment:
We choose `query` because it's not a `localization` body, but rather a generic feature (similar to `aposMode`).

Reply:
Interesting. Which APIs support this query parameter?

Reply:
The `beforeLocalize` gets this param to perform the translation if needed.